SECURITY.md

```diff
@@ -0,0 +1,25 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
```

Comment:
Releases are tags on the master branch, so it's not possible to backport patches.